src/mod10.js

```diff
@@ -0,0 +1,44 @@
+import { applyWeights, gen, validateMod, getDefaultMod10Weights } from './helpers'
+
+function mod10(digits, weights = getDefaultMod10Weights(digits.length)) {
+  const weightedDigits = applyWeights(digits.reverse(), weights)
+  const tverrsum = getTverrsum(weightedDigits)
+  return (10 - (tverrsum % 10)) % 10 == 0 ? 0 : 10 - (tverrsum % 10)
+}
+
+/**
+ * 
+ * @param {number[]} weighted weighted digits
+ */
+function getTverrsum(weighted) {
+  return weighted.reduce((acc, val) => {
+    let currentAdd = val
+    if (val > 9) {
+      currentAdd = ("" + val).split('').map(n => parseInt(n)).reduce((prev, curr) => prev + curr, 0)
+    }
+    return acc + currentAdd
+  }, 0)
+}
+
+/**
+ * 
+ * @param {number} length number of digits excluding the control digit
+ * @param {number[]} weights (optional)
```

Comment:
Mismatch between parameter and its doc

Reply:
Yes, should have correct name and be specified that is used gen as default if parameter not included

Reply:
Fixed by b2218b7